docs/developer-guide/running-substreams.md

```diff
@@ -77,3 +77,36 @@ map_transfers: message "eth.erc721.v1.Transfers": {
 ```
 
 The example output shown above contains data for different transfers from data in the blockchain. These transfers can also be verified on [Etherscan](https://etherscan.io/tx/0xcfb197f62ec5c7f0e71a11ec0c4a0e394a3aa41db5386e85526f86c84b3f2796).
+
+## Development & Production Mode
+
+### Development Mode
+
+Production and development modes impact the execution of Substreams. Key aspects of execution include:
+
+- The time required to reach the first byte.
+- The speed that large ranges get executed.
+- The module logs and outputs that are sent back to the client.
+
+Differences between production and development modes include:
+
+- Forward execution is enabled in production mode and disabled in development mode.
+- The time required to reach the first byte in development mode is faster than in production mode.
+- In development mode, the client will receive all of the executed module's logs. In production mode the client will only receive the output module's logs.
+- In development mode, it's possible to request specific store snapshots in the execution tree.
+
```

Comment:
We are also missing the fact the in development mode, multiple module's output is possible.

Reply:
Added a bullet point/sentence for multiple output. Please review for technical accuracy. It reads a bit strange to me as it stands. The changes will be available in the next commit/push.

Reply:
> It's important to remember the -p flag is passed to the substreams CLI only in development mode.

Reading this is super confusing to me. Also, it should be `Substreams CLI`.

Reply:
Sentence removed. Per the style guide when we refer to the Substreams CLI is should be formatting as:

`substreams` CLI

So, lower case here for Substreams and use the code font.